spec/requests/linking/link_defendant_maat_reference_spec.rb

```diff
@@ -52,9 +52,8 @@
         expect(flash.now[:alert]).to match(/Defendant is not a valid uuid/i)
       end
 
-      it 'redirects to defendant path' do
-        expect(response).to redirect_to new_laa_reference_path('69a73434-ae4b-4728-97b8-6a0c60f37930',
-                                                               urn: urn)
+      it 'renders laa_reference_path and does not redirect' do
```

Comment:
don't need the negative statement `and does not redirect`

Reply:
could use `expect(response).to render_template("edit")` instead

Reply:
👍 removed and does not redirect
Used expect(response).to render_template('new')